src/org/labkey/cds/CDSManager.java

```diff
@@ -330,6 +329,18 @@ public String getCDSImportFolderPath(Container container)
         return PageFlowUtil.decode(mp.getEffectiveValue(container));
     }
 
+    public String getBlogPath(Container container)
+    {
+        ModuleProperty mp = ModuleLoader.getInstance().getModule(CDSModule.class).getModuleProperties().get(CDSModule.BLOG_PATH);
+        return PageFlowUtil.decode(mp.getEffectiveValue(container));
```

Comment:
It would be nice if this method could tolerate either a relative or fully qualified URL but then return a fully qualified URL. That way the code in `frontPage.jsp` wouldn't need to concatenate prepend `baseURL` to the `blogPath`. For example the module property could be either:
- `/_webdav/home/Public/%40files/publicPage/blogs/`
- `http://localhost:8080/labkey/_webdav/home/Public/%40files/publicPage/blogs/`

but always include the base URL if necessary.

Reply:
Done